lib/Backends/NNPI/NNPI.cpp

```diff
@@ -1313,3 +1331,260 @@ Error NNPIBackend::bindContexts(
 
   return Error::success();
 }
+
+// Partial update of the NNPITensorDesc. Some members are ignored as they're
+// not used for estimation.
+static bool UpdateDescForEstimate(NNPITensorDesc &desc,
```

Comment:
nit: `updateDescForEstimate `

Reply:
nit: `const NNPITensorDesc &`

Reply:
Changed the method name but the cant change to const as desc is updated in this method.